source/docs/licensing/licensing.rst

```diff
@@ -39,6 +39,11 @@ A list of purchased (but unattached) license seats are shown here. Click on the
 
 Once the activation is complete, the license will be downloaded to the device. In the event that Tuner X disconnects from the internet or from the robot before this completes, the license is still activated and available for download the next time Tuner X is connected to the internet/robot.
 
+Activating a License without a Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Assuming that the user has connected Tuner X to the robot, the device will be available in the :guilabel:`Device History` page for :ref:`licensing while disconnected <docs/tuner/device-history:licensing from device history>`.
```

Comment:
I don't really care for the wording of "Assuming that the user has connected Tuner X to the robot, the device will be available", trying to figure out a better way to get across the idea that if Tuner X has ever been connected to the robot, all devices that were present on the robot at the time Tuner X was connected are now in the Device History page.

Reply:
"Devices that have been viewed in Tuner X at least once will be visible in Device History"?

Reply:
Replace "viewed in" with "seen by" and "visible" with "available" and I think that should be fine.